pjsip/include/pjsua-lib/pjsua_internal.h

```diff
@@ -530,7 +530,7 @@ struct pjsua_data
     /* Calls: */
     pjsua_config         ua_cfg;                /**< UA config.         */
     unsigned             call_cnt;              /**< Call counter.      */
-    pjsua_call           calls[PJSUA_MAX_CALLS];/**< Calls array.       */
+    pjsua_call           *calls;/**< Calls array.       */
```

Comment:
Cosmetics:
- the field name alignment should be done on the name, not the `*` (see other pointer fields in the struct).
- please put back the spaces for docs/comment alignment.